www/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8000",
+  "videoRecording": false
```

Comment:
According to the docs this is called `video`
https://docs.cypress.io/guides/references/configuration.html#Videos

Reply:
@LeKoArts Nice catch! Fixed. 👍